test/dymaptic.GeoBlazor.Core.Test.Blazor.Shared/Components/SymbolTests.razor

```diff
@@ -56,10 +57,10 @@
                                         VerticalAlignment="VerticalAlignment.Top"
                                         Kerning="false"
                                         LineHeight="2.3"
```

Comment:
does this need to match the adjustment that was done with linewidth? or since this is neither a point nor pixel reference...it can stay as is

Reply:
Yeah, it's a weird one, but in CSS, `line-height` is not actually pixel based by default.

> The used value is this unitless [<number>](https://developer.mozilla.org/en-US/docs/Web/CSS/number) multiplied by the element's own font size. The computed value is the same as the specified <number>. In most cases, this is the preferred way to set line-height and avoid unexpected results due to inheritance.

https://developer.mozilla.org/en-US/docs/Web/CSS/line-height